core/field_dropdown.js

```diff
@@ -234,7 +234,7 @@ Blockly.FieldDropdown.prototype.showEditor_ = function() {
   // Focusing needs to be handled after the menu is rendered and positioned.
   // Otherwise it will cause a page scroll to get the misplaced menu in
   // view. See issue #1329.
-  this.menu_.focus();
+  this.menu_.focus({preventScroll:true});
```

Comment:
This is just calling the menu's focus event not an element focus so no need here.